src/docfx/lib/JsonUtility.cs

```diff
@@ -346,7 +357,7 @@ private static bool IsNullOrUndefined(this JToken token)
                 (token.Type == JTokenType.Undefined);
         }
 
-        private static void FindNulls(this JToken token, List<Error> errors, List<JToken> nullNodes)
+        private static void RemoveNullsCore(JToken token, List<Error> errors, List<JToken> nullNodes, List<JToken> removeNodes)
```

Comment:
>List<JToken> nullNodes, List<JToken> removeNodes [](start = 78, length = 48)

seems `nullNodes` is always identical to `removeNodes`, merge into one?

Reply:
`nullNodes` contains nulls for object and array, `removeNodes` only contains nulls in arrays.